frontend/public/components/custom-resource-definition.tsx

```diff
@@ -3,11 +3,23 @@ import * as React from 'react';
 import { Link } from 'react-router-dom';
 
 import { ColHead, List, ListHeader, ListPage } from './factory';
+import { FLAGS } from '../features';
 import { Cog, ResourceCog, ResourceIcon } from './utils';
 import { referenceForCRD } from '../module/k8s';
+import { promoteCrdModal } from './modals';
 
 const { common } = Cog.factory;
-const menuActions = [...common];
+const PromoteCRD = (kind, crd) => ({
+  label: 'Promote Custom Recource Definition',
```

Comment:
I would call this something like "Add to Console Navigation". "Promote" is too general a word I think.